lms/templates/edraak_forus/auth.html

```diff
@@ -0,0 +1,38 @@
+<%page expression_filter="h"/>
```

Comment:
Still not sure that this has been migrated. We agreed to do one of the following approaches:

 - Add some kind of tests to ensure only needed modifications are done.
 - Refactor into three files: a) Base, b) edX auth, c) ForUs auth.